Minesweeper.py

```diff
@@ -99,6 +99,8 @@ def game_loop():
             if board[player_pos[1]][player_pos[0]] == 0:
                 board[player_pos[1]][player_pos[0]] = "F"
                 flags += 1
+            elif board[player_pos[1]][player_pos[0]] == "F":  # Error: Redundant condition, already checked in unflag
```

Comment:
There is a redundant condition that has already been checked in the previous conditional statement. This checks for the same condition again, which can be simplified.
```diff
-            elif board[player_pos[1]][player_pos[0]] == "F":  # Error: Redundant condition, already checked in unflag
+            # Remove this redundant check for "F"
```
---